source/fundamentals/auth.txt

```diff
@@ -389,10 +389,21 @@ parameter as shown in the following example:
 The next examples show how to provide your credentials by setting environment
 variables for the following types of authentication:
 
+- Web identity provider
 - Programmatic access keys
 - ECS container credentials
 - EC2 container credentials
 
+You can use an OpenID Connect (OIDC)-compatible **web identity provider** to authenticate
+to Amazon Elastic Kubernetes Service (EKS) or other services.
+To use a web identity provider, create a file that contains your
+OIDC token, then set the absolute path to this file in an environment variable by using
+``bash`` or a similar shell as shown in the following example:
```

Comment:
[suggestion]
I found this too long to easily parse and it's over our suggested word length. I'd consider chopping this up into two sentences to make it easier to understand.

```suggestion
To use a web identity provider, create a file that contains your
OIDC token. Then set an environment variable to the absolute path to this file by using
``bash`` or a similar shell as shown in the following example:
```